### PR TITLE
Rename Cesium3DTiles -> Cesium3DTilesSelection

### PR DESCRIPTION
**Pull request by krupkad**
Renaming `Cesium3DTiles` to `Cesium3DTilesSelection` to make room for 3D Tiles serialization under the current naming scheme (`Cesium3DTiles`, `Cesium3DTilesReader` and `Cesium3DTilesWriter`)

### CODE REVIEW

File: Cesium3DTilesSelection/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
+add_library(Cesium3DTilesSelection "")
+
+configure_cesium_library(Cesium3DTilesSelection)
+
+cesium_glob_files(CESIUM_3DTILES_PIPELINE_SOURCES src/*.cpp)
```
Comment (kring) [inline Cesium3DTilesSelection/CMakeLists.txt:5]: PIPELINE vs SELECTION?

File: Cesium3DTilesSelection/include/Cesium3DTilesSelection/Library.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+/**
+ * @brief Classes that implement the 3D Tiles standard
+ */
+namespace Cesium3DTilesSelection {}
+
+#if defined(_WIN32) && defined(CESIUM_SHARED)
+#ifdef CESIUM3DTILES_BUILDING
```
Comment (kring) [inline Cesium3DTilesSelection/include/Cesium3DTilesSelection/Library.h:9]:
```suggestion
#ifdef CESIUM3DTILESSELECTION_BUILDING
```

File: Cesium3DTilesSelection/test/data/Tileset/README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
 The contents of this directory are from CesiumJS:
-https://github.com/CesiumGS/cesium/tree/4f0d17c/Apps/SampleData/Cesium3DTiles/Tilesets/Tileset
+https://github.com/CesiumGS/cesium/tree/4f0d17c/Apps/SampleData/Cesium3DTilesSelection/Tilesets/Tileset
```
Comment (kring) [inline Cesium3DTilesSelection/test/data/Tileset/README.md:2]: This rename is incorrect; it's a link into the CesiumJS repo.

**Review by kring — COMMENTED** [review]
Noticed a few small things, but looks good overall.

**Comment by baothientran** [comment]
This looks good to me too!

**Comment by lilleyse** [comment]
@baothientran thanks for opening https://github.com/CesiumGS/cesium-unreal/pull/582.

Should this PR be merged first and then pull the changes into Unreal? I think that would be the standard workflow.

**Comment by baothientran** [comment]
yeah I don't see any problems with merge this PR first

**Comment by lilleyse** [comment]
Thanks for merging these PRs @baothientran.

@krupkad can you open an issue detailing the work left?

**Pull request merged**